pypsa/plot.py

```diff
@@ -360,55 +368,62 @@ def projected_area_factor(ax, original_crs):
 
 
 def draw_map_basemap(network, x, y, ax, boundaries=None, margin=0.05,
-                     geomap=True, basemap_parameters=None):
+                     geomap=True, basemap_parameters=None, color_geomap=None):
 
     if boundaries is None:
         (x1, y1), (x2, y2) = compute_bbox_with_margins(margin, x, y)
     else:
         x1, x2, y1, y2 = boundaries
 
-    #First choice should be cartopy
-    if cartopy_present:
-        resolution = '50m' if isinstance(geomap, bool) else geomap
-        assert resolution in ['10m', '50m', '110m'], (
-                "Resolution has to be one of '10m', '50m', '110m'")
-        gmap = ax.projection
-        data_projection = get_projection_from_crs(network.srid)
-        ax.set_extent([x1, x2, y1, y2], crs=data_projection)
-        if color_geomap:
-            ax.add_feature(cartopy.feature.OCEAN.with_scale(resolution),
-                           facecolor='lightblue')
-            ax.add_feature(cartopy.feature.LAND.with_scale(resolution),
-                           facecolor='whitesmoke')
-        ax.coastlines(linewidth=0.4, zorder=-1, resolution=resolution)
-        border = cartopy.feature.BORDERS.with_scale(resolution)
-        ax.add_feature(border, linewidth=0.3)
-
-    elif basemap_present:
-        resolution = 'l' if isinstance(geomap, bool) else geomap
-        gmap = Basemap(resolution=resolution,
-                       llcrnrlat=y1, urcrnrlat=y2, llcrnrlon=x1,
-                       urcrnrlon=x2, ax=ax)
-        gmap.drawcountries(linewidth=0.3, zorder=-1)
-        gmap.drawcoastlines(linewidth=0.4, zorder=-1)
-        if color_geomap:
-            gmap.drawlsmaks(land_color="whitesmoke",
-                            ocean_color="lightblue",
-                            grid=1.25, ax=ax)
-        # no transformation -> use the default
-        data_projection = ax.transData
-
-        # disable gmap transformation due to arbitrary conversion
-        # x, y = gmap(x.values, y.values)
-
-    return data_projection
+    if basemap_parameters is None:
+        basemap_parameters = {}
+
+    resolution = 'l' if isinstance(geomap, bool) else geomap
+    gmap = Basemap(resolution=resolution,
+                    llcrnrlat=y1, urcrnrlat=y2, llcrnrlon=x1,
+                    urcrnrlon=x2, ax=ax, **basemap_parameters)
+    gmap.drawcountries(linewidth=0.3, zorder=-1)
+    gmap.drawcoastlines(linewidth=0.4, zorder=-1)
+
+    if color_geomap is None:
+        color_geomap = {'ocean': 'w', 'land': 'w'} 
```

Comment:
Intentional to use the same color for `land` and `ocean`? You did use `whitesmoke` and `lightblue` before.

Reply:
Yes. If one doesn’t specify colours, the map remains in the “minimal colour setting” as it was before.